src/Plugin/Field/FieldWidget/ProductVariationOptionsWidget.php

```diff
@@ -192,6 +192,12 @@ public function formElement(
       '#attributes' => [
         'class' => ['option-widgets'],
       ],
+      '#prefix' => '<div class = "product__internal">'
```

Comment:
Can you rename this class to "product__commerce_options__pricing_note" or something. Also, can you provide a screenshot of where it appears.